Themis/Views/Custom/DateTimelineView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  TimelineView.swift
+//  Themis
+//
+//  Created by Katjana Kosic on 02.02.23.
+//
+
+import SwiftUI
+
+struct DateTimelineView: View {
+    let dates: [(name: String, date: String?)]
+    @State private var showingAlert = false
+    @State private var selectedDate: (String, String?) = ("", nil)
+
+    var body: some View {
+        ZStack(alignment: .center) {
+            Rectangle()
+                .frame(width: 500, height: 5)
+                .foregroundColor(Color.blue)
+                .cornerRadius(5)
+            HStack {
+                ForEach(dates, id: \.name) { date in
+                    VStack {
+                        Text(date.name).bold()
+                        Image(systemName: "calendar")
+                            .background(Color(.systemBackground))
+                            .font(.title3)
+                            .padding(1)
+                            .onLongPressGesture {
+                                selectedDate = (date.name, date.date)
+                                showingAlert = true
+                            }
+                        if let dateString = ArtemisDateHelpers.getRemainingOrOverdueTime(for: date.date) {
+                            Text(dateString)
+                        } else {
+                            Text("Not available yet")
+                        }
+                    }
+                    .frame(maxWidth: 150)
+                    .padding(2)
+                }
+            }
+        }
+        .alert(isPresented: $showingAlert) {
```

Comment:
What does this alert do?

Reply:
It enables users to long press on a date to know the exact date, however it is still a work in progress as the popup is not showing up exactly above the pressed date.
![simulator_screenshot_B73748B2-E2C7-4CB8-B691-1BD3BF9101D4](https://user-images.githubusercontent.com/24537483/217224114-a515fb38-0c8b-4a1b-9002-cda2bac36c57.png)

Reply:
Ah, now I understand. Can you write "date" in with a capital letter -> "Date"?

Reply:
It there is no date, it does not say that it is not available yet.

<img width="807" alt="image" src="https://user-images.githubusercontent.com/116847304/217230614-ccc660b4-c050-4726-9cf4-07bed2cbc9f9.png">

Reply:
Since this feature will be removed for now, I'll mark this conversation as resolved.